src/plugins/intel_gpu/include/intel_gpu/primitives/experimental_detectron_prior_grid_generator.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "primitive.hpp"
+
+namespace cldnn {
+struct experimental_detectron_prior_grid_generator: public primitive_base<experimental_detectron_prior_grid_generator> {
+    CLDNN_DECLARE_PRIMITIVE(experimental_detectron_prior_grid_generator)
+
```

Comment:
Please add comment block for doxygen documentation.
```suggestion
/// @brief Constructs experimental_detectron_prior_grid_generator primitive.
/// @param id This primitive id
…
```

Reply:
done (in PR https://github.com/openvinotoolkit/openvino/pull/11632 )